Integrations/integration-Zendesk.yml

```diff
@@ -511,17 +522,23 @@ script:
             ec.Ticket = newContext;
         }
 
-
         var res = {
             TicketDetails: resDetails,
             TicketComments: resComments,
             Users: usersEc,
+            Org: org,
             CustomFields: fieldsEc
         }
-
         return({ContentsFormat: formats.json, Type: entryTypes.note, Contents: res, HumanReadable: md, EntryContext: ec})
     };
 
+    var getArticle = function() {
+        var locale = (args.locale) ?  args.locale : 'en-us';
+        var id = args.articleID;
```

Comment:
no need for this assignment, just use it after in the URI

Reply:
fixed